docs/spec/wire.md

```diff
@@ -193,8 +193,9 @@ Further non-successful status codes are defined in the Conjure errors section be
 ### 3.2. Response body
 Conjure servers must serialize return values using the [JSON format][] defined below, unless:
 
-  - the de-aliased return type is `optional<T>` and the value is not present: servers must omit the HTTP body.
-  - the de-aliased return type is `binary`: servers must write the raw bytes as an octet-stream.
+  - the de-aliased return type is `optional<T>` and the value is not present: servers must return an empty HTTP body
```

Comment:
I think we should specify 204 response code, and no http response body

Reply:
the 204 thing is defined above. "empty body" is equivalent to "no body", right?

Reply:
It should be, though an empty body could be construed to have a content type, where no body would not, and aligns with our “no content” response code.

I’m happy either way